spec/README.md

```diff
@@ -72,8 +72,12 @@ A **_term_** looks like this when it is defined in this specification.
 
 A reference to a _term_ looks like this.
 
+> [!NOTE]
 > Examples are non-normative and styled like this.
```

Comment:
This change ought to be reflected in the text. We have _many_ `>` indented blocks without a `[!NOTE]` specifier that are currently non-normative, and which would be made maybe-normative by this change.

Reply:
Agreed. I dithered about adding the `[!NOTE]` notation. I will remove it and add a separate bit about notes.